vcr/interface.go

```diff
@@ -60,7 +60,7 @@ var revocationDocumentType = "application/vc+json;type=revocation"
 
 // ConceptFinder can resolve VC backed concepts for a DID.
 type ConceptFinder interface {
-	// Get returns the requested concept for the subject or ErrNotFound
+	// Get returns the requested concept as JSON string for the subject or ErrNotFound
```

Comment:
it returns it as concept, which is a `map[string]interface{}`? (not as JSON string)

Reply:
ahhh yes, I wanted it to be a string but it didn't work out.